.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: RFE
+assignees: ''
+
+---
+
+If your feature need figuring out how to implement it or needs feedback from the wider comunity, please open a [Discussion](https://github.com/rpm-software-management/rpm/discussions) instead. If the discussion has solidified into a plan of action it is time to create an issue for actually implementing it.
```

Comment:
I think we'll want a stronger emphasis on the discussions. Basically: *always* start a discussion first, using the template info below as the starter, and: "Once the discussion has solidifed into a plan of action, the rpm maintainers will open an issue (or request you to do it) for actually implementing it.".

Reply:
I don't think that's necessary, since Issues can be converted into Discussions and back.

Reply:
It's manual work that, ahem, somebody gets to do. And the average pony request is better off starting as a discussion, to avoid the overhead that goes into ticket processing.

Reply:
Unless someone is committing to doing that same work in Discussions, we should not be telling people to go there. Discussions should not be treated like that.

Reply:
It's not the same work, otherwise there would be little point in all this. Discussions are better for discussions than tickets, and tickets are better for organizing work.

This is work *we* do so maybe we get to decide how to do it, no?

Reply:
Just FTR, I dislike the discussions, mainly because I read everything through rpm-maint ML and the discussions are not forwarded there AFAICT. And I don't know if there is any meaningful way to subscribe to discussions, but I don't see any on the first look. IOW they might look like the best tool for the team or project members, but IMHO, they prevent collaboration in wider community.

Reply:
Oh, we have a confirmed rpm-maint reader. I've suspected nobody looks at that anymore because volume and format...

Reply:
> Just FTR, I dislike the discussions, mainly because I read everything through rpm-maint ML and the discussions are not forwarded there AFAICT.

Hm, checking the archives, I was wrong. I might change my mind, sorry for the nose.

Reply:
Ack, I thought they should be forwarded along with all the other traffic. And, if not then it should at least be possible.

Reply:
> Oh, we have a confirmed rpm-maint reader. I've suspected nobody looks at that anymore because volume and format...

At least my rumbling was good for something :see_no_evil: Yes, rpm-maint is the most consumable format allowing to follow what is going on with RPM.